src/ua/dymohlo/parametersvaliditycheck/ValidValues.java

```diff
@@ -0,0 +1,7 @@
+package ua.dymohlo.parametersvaliditycheck;
+
+public class ValidValues extends Exception {
```

Comment:
Please use more appropriate name for this exception 
For example InvalidValuesException